ui-tests/src/test/java/org/openmrs/reference/TransferToWardServiceTest.java

```diff
@@ -13,13 +13,15 @@
 import org.junit.Before;
 import org.junit.Ignore;
 import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.openmrs.reference.groups.BuildTests;
 import org.openmrs.reference.page.ClinicianFacingPatientDashboardPage;
 import org.openmrs.uitestframework.test.TestData;
 
 import static org.junit.Assert.assertTrue;
 
-
-public class TransferToWardServiceTest extends ReferenceApplicationTestBase {
+@Ignore
+public class TransferToWardServiceTest extends LocationSensitiveApplicationTestBase {
```

Comment:
Any reason why you changed it to LocationSensitiveApplicationTestBase

Reply:
After fixing issues with this particular test `TrasferToWardServiceTest` and  `code-documentation` I will then migrate the ticket state to `READY FOR REVIEW`  for reviewers to take a glace at the fix